build.gradle

```diff
@@ -178,16 +178,18 @@ dependencies {
     compile group: 'com.github.hmcts', name: 'idam-java-client', version: '2.0.1'
     compile "org.bouncycastle:bcprov-jdk15on:1.61", "org.bouncycastle:bcpkix-jdk15on:1.61"
 
-    implementation group: 'org.springframework.security', name: 'spring-security-oauth2-client'
+    implementation group: 'org.springframework.security', name: 'spring-security-oauth2-client', version: '5.6.1'
```

Comment:
As the version is getting repeated in many places, can you move it to ext above and refer the variables here?

Reply:
Agree.